### PR TITLE
chore:  Update Lambda Runtime to NODEJS_12_X

### PR DESCRIPTION
**Pull request by curry9999**
Fixes # 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: csharp/my-widget-service/src/MyWidgetService/MyWidgetService.csproj [hunk 1]
```diff
@@ -6,10 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Amazon.CDK" Version="1.14.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.ApiGateway" Version="1.14.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Lambda" Version="1.14.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.S3" Version="1.14.0-devpreview" />
+    <PackageReference Include="Amazon.CDK" Version="1.18.0-devpreview" />
```
Comment (rhboyd) [inline csharp/my-widget-service/src/MyWidgetService/MyWidgetService.csproj]: There is no `1.18.0-devpreview` package it should just say `1.18.0`

File: csharp/random-writer/src/RandomWriter/RandomWriter.csproj [hunk 2]
```diff
@@ -6,11 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Amazon.CDK" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.DynamoDb" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Events" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Events.Targets" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Lambda" Version="1.15.0-devpreview" />
+    <PackageReference Include="Amazon.CDK" Version="1.18.0-devpreview" />
```
Comment (rhboyd) [inline csharp/random-writer/src/RandomWriter/RandomWriter.csproj]: same as the other comment, there is no `1.18.0-devpreview` package

Comment (curry9999) [reply csharp/random-writer/src/RandomWriter/RandomWriter.csproj]: Thanks!

File: csharp/my-widget-service/src/MyWidgetService/MyWidgetService.csproj (hunk 1 above)
Comment (curry9999) [reply csharp/my-widget-service/src/MyWidgetService/MyWidgetService.csproj]: Thanks!

**Comment by NGL321** [comment]
Looks like package-references still say 1.18.0-devpreview. Also, it seems that the base-repo is gone.

@curry9999 Any status update?

**Pull request closed without merging**